notify.rb

```diff
@@ -0,0 +1,23 @@
+APP_ROOT = File.dirname(__FILE__)
+
+$:.unshift(File.join(APP_ROOT, 'lib'))
+require 'birthday_bot'
+require 'config_reader'
+
+puts '🤖 Bot is booting...'
+
+configs = ConfigReader.new()
+if not configs.load('configurations.json')
+  abort '❌ Bot is shutting down'
+end
+
+puts 'Reading configurations'
+url = configs.slack_url
+channel = configs.channel_name
+greeting = configs.greeting_msg
+name = configs.bot_name
+emoji = configs.bot_emoji
+
+puts '🤖 Bot is activating...'
+bot = SlackBot.new(url, channel, greeting, name, emoji)
```

Comment:
@dialex Instead of generating duplicate variables for configurations we could just pass the `configs` object to the `SlackBot` constructor with all the configuration info.